packages/view/backend/sirius-components-view-diagram/src/main/java/org/eclipse/sirius/components/view/diagram/impl/EdgeStyleImpl.java

```diff
@@ -759,31 +794,32 @@ public String toString() {
         if (this.eIsProxy())
             return super.toString();
 
-        StringBuilder result = new StringBuilder(super.toString());
-        result.append(" (fontSize: ");
-        result.append(this.fontSize);
-        result.append(", italic: ");
-        result.append(this.italic);
-        result.append(", bold: ");
-        result.append(this.bold);
-        result.append(", underline: ");
-        result.append(this.underline);
-        result.append(", strikeThrough: ");
-        result.append(this.strikeThrough);
-        result.append(", lineStyle: ");
-        result.append(this.lineStyle);
-        result.append(", sourceArrowStyle: ");
-        result.append(this.sourceArrowStyle);
-        result.append(", targetArrowStyle: ");
-        result.append(this.targetArrowStyle);
-        result.append(", edgeWidth: ");
-        result.append(this.edgeWidth);
-        result.append(", showIcon: ");
-        result.append(this.showIcon);
-        result.append(", labelIcon: ");
-        result.append(this.labelIcon);
-        result.append(')');
-        return result.toString();
+        String result = super.toString() + " (fontSize: " +
+                this.fontSize +
+                ", italic: " +
+                this.italic +
+                ", bold: " +
+                this.bold +
+                ", underline: " +
+                this.underline +
+                ", strikeThrough: " +
+                this.strikeThrough +
+                ", lineStyle: " +
+                this.lineStyle +
+                ", sourceArrowStyle: " +
+                this.sourceArrowStyle +
+                ", targetArrowStyle: " +
+                this.targetArrowStyle +
+                ", edgeWidth: " +
+                this.edgeWidth +
+                ", showIcon: " +
+                this.showIcon +
+                ", labelIcon: " +
+                this.labelIcon +
+                ", maxWidthExpression: " +
+                this.maxWidthExpression +
+                ')';
+        return result;
```

Comment:
Why the generated code switched from string builder to string concatenation?

Reply:
It's my formatter that transforms this code.
It seems to be the same for others generated classes